lib/config.js

```diff
@@ -152,6 +153,22 @@ const conf = convict({
       default: 10
     }
   },
+  openid: {
+    key: {
+      doc: 'Private JWK to sign id_tokens',
+      default: {}
+    },
```

Comment:
@jrgm hi there! how would you prefer we load a secret key for signing JWTs? This currently has it being in JSON form in the config, but is a file better?